### ISSUE TITLE
Auto deployment Using Github Actions

### ISSUE THREAD

**Issue by Zemill** [Heroes-Profile/heroesprofile#67]
I would like to be able to publish the app to GAE/Google Kubernetes using github actions.  Both auto deployment, and manual deployment.

I am not sure if this is entirely possible, but creating a feature request so I do not forget, and if anyone has experience doing it.

**Comment by MGatner** [comment]
This is definitely doable! I don’t have a lot of Kubernetes experience but I’ve been using Actions for a few months. This looks like maybe a good starting point:

https://blog.digitalocean.com/how-to-deploy-to-digitalocean-kubernetes-with-github-actions/

**Comment by Zemill** [comment]
@poppahorse I plan to take a look at the reference document @MGatner posted above.  But this seems like something you have some knowledge in as well.

**Comment by poppahorse** [comment]
Yes definitely, once we get the kubernetes stuff sorted this was next on my list. @MGatner say hi on discord one evening and we'll have a talk!
